content/en/docs/tutorials/backup.md

```diff
@@ -49,5 +57,114 @@ created above after installing cert-manager.
 $ kubectl apply -f cert-manager-backup.yaml
 ```
 
-If you have migrated from an old cluster, you will need to make sure to run a
-similar `kubectl apply` command to restore your Secret resources too.
+## Full cluster backup and restore
+
+This section refers to backing up and restoring 'all' Kubernetes resources in a
+cluster- including some `cert-manager` ones- for scenarios such as disaster
+recovery, cluster migration etc.
+
+Note that we have not done any extensive testing, only a few common scenarios-
+it is recommended to test the backup and restore strategy before putting it in place
+as there are likely to be some other edge cases.
+We would like to ensure that users can reliably backup and restore `cert-manager`
+resources using common tools- if you encounter any errors, please do open a
+GitHub issue or PR an update to this document.
```

Comment:
```suggestion
*Note*: We have tested this process on simple Kubernetes test clusters with a limited set of Kubernetes releases. To avoid data loss, please test both the backup and the restore strategy on your own cluster before depending upon it in production. If you encounter any errors, please open a GitHub issue or a PR to document variations on this process for different Kubernetes environments. 
```

I suggest something like that ^^  instead, but happy to go with the original version if you prefer.